test/e2e/framework/framework.go

```diff
@@ -115,6 +116,7 @@ func NewFromFlags() (*Framework, error) {
 		"optional secret to be used for pulling the gameserver and/or Agones SDK sidecar images")
 	stressTestLevel := flag.Int("stress", 0, "enable stress test at given level 0-100")
 	perfOutputDir := flag.String("perf-output", "", "write performance statistics to the specified directory")
+	version := flag.String("version", "", "version defaults to the short hash of the latest commit")
```

Comment:
Version is usually in form `1.5.0-89abcdef` and is defaulted in Make target not here. So it is better to say: 
```suggestion
	version := flag.String("version", "", "Agones controller version which was tested, in the form of release version plus the short hash of the latest commit")
```

Reply:
Tnanks for the suggestion